### ISSUE TITLE
[qt5-imageformats:x64-windows-static] Building package qt5-imageformats:x64-windows-static failed with: BUILD_FAILED

### ISSUE THREAD

**Issue by jabra98** [microsoft/vcpkg#10956]
**Host Environment**
- OS: Windows 10
- Compiler: latest version of MSVC

**To Reproduce**
`./vcpkg install qt5:x64-windows-static`

**Failure logs** 
```-- Using cached D:/Libs/vcpkg/downloads/qtimageformats-everywhere-src-5.12.5.tar.xz
-- Extracting source D:/Libs/vcpkg/downloads/qtimageformats-everywhere-src-5.12.5.tar.xz
-- Using source at D:/Libs/vcpkg/buildtrees/qt5-imageformats/src/5.12.5-7ddf5dd29e
-- Configuring x64-windows-static-rel
-- Configuring x64-windows-static-rel done
-- Configuring x64-windows-static-dbg
-- Configuring x64-windows-static-dbg done
-- Package build-x64-windows-static-dbg
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:72 (message):
    Command failed: D:/Libs/vcpkg/downloads/tools/jom/jom-1.1.3/jom.exe
    Working Directory: D:/Libs/vcpkg/buildtrees/qt5-imageformats/x64-windows-static-dbg
    Error code: 2
    See logs for more information:
      D:\Libs\vcpkg\buildtrees\qt5-imageformats\package-build-x64-windows-static-dbg-out.log
      D:\Libs\vcpkg\buildtrees\qt5-imageformats\package-build-x64-windows-static-dbg-err.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_build_qmake.cmake:32 (vcpkg_execute_required_process)
  scripts/cmake/vcpkg_build_qmake.cmake:67 (run_jom)
  installed/x64-windows-static/share/qt5/qt_build_submodule.cmake:12 (vcpkg_build_qmake)
  installed/x64-windows-static/share/qt5/qt_submodule_installation.cmake:8 (qt_build_submodule)
  ports/qt5-imageformats/portfile.cmake:39 (qt_submodule_installation)
  scripts/ports.cmake:90 (include)


Error: Building package qt5-imageformats:x64-windows-static failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: qt5-imageformats:x64-windows-static
  Vcpkg version: 2020.02.04-nohash-external
```

Additionally, attach any relevant sections from the log files above.
[package-build-x64-windows-static-dbg-err.log](https://github.com/microsoft/vcpkg/files/4517500/package-build-x64-windows-static-dbg-err.log)
[package-build-x64-windows-static-dbg-out.log](https://github.com/microsoft/vcpkg/files/4517501/package-build-x64-windows-static-dbg-out.log)

**Additional context**
It actually compiles when using the dynamic triplet (the one without the -static suffix).
I have built other packages with the `x64-windows-static` triplet, so that shouldn't be the issue

**Comment by Neumann-A** [comment]
fixed in #10644

**Comment by Neumann-A** [comment]
@jabra98 closed?
